pkgs/applications/networking/modem-manager-gui/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, buildEnv, pkgconfig, python3, fetchhg, gtk3-x11, glib, gdbm, gtkspell3, itstool, libappindicator-gtk3, meson, ninja, perldevelPackages }:
+
+stdenv.mkDerivation rec {
+  name = "modem-manager-gui";
+  version = "0.4.17";
+  rev = "v${version}";
+
+  src = fetchhg {
+    url = https://linuxonly@bitbucket.org/linuxonly/modem-manager-gui;
+    rev = "8bcf842";
+    sha256 = "04p08gzvi068bd3slbnaghajwzp4h5jacib640k5shngqimkl84p";
+  };
+
+  buildInputs = [
+    pkgconfig
+    gtk3-x11
+    glib
+    gdbm
+    gtkspell3
+
+    python3
+    perldevelPackages.Po4a
+    itstool
+    libappindicator-gtk3
+  ];
+
+  preConfigure = ''
+    sed -i 's|! test -e "/usr/include/gdbm/gdbm.h"|false|' configure
```

Comment:
Could you create a patch file instead? `sed` calls get outdated pretty fast without any warning.